docs/pages/dev-guide/developer-guide.md

```diff
@@ -1254,21 +1256,21 @@ pods:
 ```
 
 
-* `EXTERNAL_ROUTES` and `INTERNAL_ROUTES`
+1. Delete these 3 labels from your marathon json:
+    * `DCOS_FRAMEWORK_NAME`
+    * `DCOS_SERVICE_PORT_INDEX`
+    * `DCOS_SERVICE_SCHEME`
+
+1. `EXTERNAL_ROUTES` and `INTERNAL_ROUTES`
 
     * These have a 1:1 mapping (they are both comma separated lists). There is one internal route for every external route.
 
     * For example, in the declaration above, if you navigate to `<adminrouter>/service/{{FRAMEWORK_NAME}}/v1/plan`, you’ll get redirected to `{{FRAMEWORK_NAME}}.marathon.mesos:{{PORT0}}/v1/plan`
 
-* `ROOT_REDIRECT`
+1. `ROOT_REDIRECT`
 
     * This will set a redirect from `/` (a.k.a. the root path) to a path of your choosing. For example, `/example` redirects `<adminrouter>/service/{{FRAMEWORK_NAME}}` to `<adminrouter>/service/{{FRAMEWORK_NAME}}/example`
 
-1. Delete these 3 labels from your marathon json:
-    * `DCOS_FRAMEWORK_NAME`
-    * `DCOS_SERVICE_PORT_INDEX`
-    * `DCOS_SERVICE_SCHEME`
-
 1. Things to watch out for:
     *  No trailing slashes.
```

Comment:
Hmm I don't think there is a way to tell that they've misconfigured it, the best we can do is give them something to try when they are debugging.